src/IntersonArrayCxxImagingContainer.cxx

```diff
@@ -341,10 +349,10 @@ class ContainerImpl
     void *clientData = 0 )
   {
     this->RFHandler->SetNewRFImageCallback( callback, clientData );
-    std::cout << "Callback width = " << this->GetWidthScan() << std::endl;
-    std::cout << "        height = " << this->GetHeightScan() << std::endl;
-    this->RFHandler->SetImageSize( this->GetWidthScan(),
-      this->GetHeightScan() );
+    std::cout << "Callback width = " << Container::NBOFLINES << std::endl;
+    std::cout << "        height = " << Container::MAX_RFSAMPLES << std::endl;
+    this->RFHandler->SetImageSize(Container::NBOFLINES,
```

Comment:
The looping in the handler might be wrong - it might be transposing the image - or the order of these args needs to be changed for consistency.